docs/sdks/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 9.0.0
+
+* Add offline support
```

Comment:
There should be other changes from Appwrite 1.3.x so
```
* updated to support Appwrite 1.3.x
* detailed changelog (link to appwrite's changelog
```

Reply:
Of course. That can be added next once the other stuff is added in. 

> Also, I believe GETTING_STARTED.md should also be updated with changes regarding the offline support, how to enable, disable it, set cache size and stuff

Should we publish that yet considering offline support is beta/experimental?

Reply:
@lohanidamodar, I've updated the changelog a bit. It may need to be updated more after relationships is merged in.

As for offline support docs in the README, I think it would be better for people to look at the offline docs on our website for now.

Reply:
We can add a section as `experimental`, most of the time Flutter/Dart people are looking at readme in `pub.dev` so it might be a good idea. WDYT?